scripts/apps/authoring/versioning/history/history.js

```diff
@@ -58,7 +60,10 @@ function HistoryController($scope, authoring, api, notify, desks, archiveService
 VersioningHistoryDirective.$inject = [];
 function VersioningHistoryDirective() {
     return {
-        templateUrl: 'scripts/apps/authoring/versioning/history/views/history.html'
+        templateUrl: 'scripts/apps/authoring/versioning/history/views/history.html',
+	scope: {
+            item: '='
```

Comment:
This is needed when an Item is previewed from Legal Archive page ( followed design similar to sd-media-metadata directive ).